packaging/deb/ogon-channels/copyright

```diff
@@ -0,0 +1,24 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ogon-channels
+Upstream-Contact: ogon <ogon@googlegroups.com>
+Source: http://github.com/ogon/ogon-channels
+
+Files: *
+Copyright: 2018 Thincast Technologies GmBH
+License: Apache-2.0
```

Comment:
That's the wrong license. ogon-channels is AGPL 3.0 with OGON CORE AGPL EXCEPTIONS see LICENSE file